AllReadyApp/Web-App/AllReady/Views/Activity/Activity.cshtml

```diff
@@ -7,6 +7,8 @@
     var userTimeZoneId = User.GetTimeZoneId();
 }
 
+<div id="alert_placeholder"></div>

```

Comment:
@mheggeseth Yes, I agree. I had thought about that at one time, but forgot to come back to it.  I'll work on some changes.